ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateSignatureProvider.fs

```diff
@@ -192,7 +192,12 @@ type FSharpGenerateSignatureBuilder() =
                 factory.CreateTypeMemberSignature(sigStrings)
             | :? IExceptionDeclaration as exceptionDeclaration ->
                 let sigExceptionDeclaration = exceptionDeclaration.Copy()
-                if not (Seq.isEmpty exceptionDeclaration.MemberDeclarations) then
+                let memberDeclarations =
+                    exceptionDeclaration.MemberDeclarations
+                    |> Seq.filter (function | :? IExceptionFieldDeclaration -> false | _ -> true)
+                    |> Seq.toArray
```

Comment:
`exceptionDeclaration.TypeMember` is probably what you're after here.

Reply:
Yup